EasyModbusSource/LICENSE/apache-2.txt

```diff
@@ -0,0 +1,203 @@
+
```

Comment:
i copied it from the JDBC extension source . do you mean remove the apache-2.txt from the licenses ?

Reply:
Well, there should be license for each the SW packages that we include.  The EasyModbus we'll have them download, so we don't have to include that.  If there's no other thing for which the Apache license is applicable, then remove it.  Otherwise, keep it consistent with the other names.